scripts/docker/official/nginx-oss-with-nginx-agent/alpine/Dockerfile.mainline

```diff
@@ -3,7 +3,7 @@ ARG NGINX_VERSION=1.25.1-alpine
 FROM nginx:${NGINX_VERSION}
 LABEL maintainer="NGINX Agent Maintainers <agent@nginx.com>"
 
-ARG NGINX_AGENT_VERSION=2.26.1
```

Comment:
should we pull the latest tag from the agent repo for this value?

Reply:
No so the Dockerfiles need to be consistent everytime we build them. For example if we release v2.28.0 of the agent and I want to build the v2.27.0 Dockerfile again then we won't be able to since it will pull the latest agent (v2.28.0)